scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv.py

```diff
@@ -0,0 +1,242 @@
+                 'CALicensedHealthcareFacilityType', 'CALicensedHealthcareFacilityBedType',
+                 'CALicensedHealthcareFacilityBedCapacity']
+
+FDR_to_type = {
+    'SKILLED NURSING FACILITY': 'SkilledNursingFacility',
+    'INTERMEDIATE CARE FACILITY-DD/H/N/CN/IID': 'IntermediateCareFacility',
+    'CONGREGATE LIVING HEALTH FACILITY': 'CongregateLivingHealthFacility',
+    'INTERMEDIATE CARE FACILITY': 'IntermediateCareFacility',
+    'HOSPICE FACILITY': 'HospiceFacility',
+    'GENERAL ACUTE CARE HOSPITAL': 'GeneralAcuteCareFacility',
+    'ACUTE PSYCHIATRIC HOSPITAL': 'AcutePsychiatricFacility',
+    'HOSPICE': 'HospiceFacility',
+    'PEDIATRIC DAY HEALTH & RESPITE CARE FACILITY': 'PediatricDayHealthAndRespiteCareFacility',
+    'CHRONIC DIALYSIS CLINIC': 'ChronicDialysisFacility',
+    'CHEMICAL DEPENDENCY RECOVERY HOSPITAL': 'ChemicalDependencyRecoveryFacility',
+    'CORRECTIONAL TREATMENT CENTER': 'CorrectionalTreatmentCenterFacility'
+}
+
+bed_type_dict = {
+    'SKILLED NURSING': 'SkilledNursingBed',
+    'SPECIAL TREATMENT PROGRAM': 'SpecialTreatmentProgramBed',
+    'INTERMEDIATE CARE/DD HABILITATIVE': 'IntermediateCareHabilitativeBed',
+    'INTERMEDIATE CARE/DD NURSING': 'IntermediateCareNursingBed',
+    'CONGREGATE LIVING HEALTH FACILITY': 'CongregateLivingHealthFacilityBed',
+    'INTERMEDIATE CARE': 'IntermediateCareBed',
+    'HOSPICE': 'HospiceBed',
+    'CORONARY CARE': 'CoronaryCareBed',
+    'INTENSIVE CARE': 'IntensiveCareBed',
+    'INTENSIVE CARE NEWBORN NURSERY': 'IntensiveCareNewbornNurseryBed',
+    'PERINATAL': 'PerinatalBed',
+    'UNSPECIFIED GENERAL ACUTE CARE': 'UnspecifiedGeneralAcuteCareBed',
+    'BURN': 'BurnBed',
+    'PEDIATRIC': 'PediatricBed',
+    'RENAL TRANSPLANT': 'RenalTransplantBed',
+    'REHABILITATION': 'RehabilitationBed',
+    'ACUTE RESPIRATORY CARE': 'AcuteRespiratoryCareBed',
+    'ACUTE PSYCHIATRIC CARE': 'AcutePsychiatricCareBed',
+    'CHEMICAL DEPENDENCY RECOVERY': 'ChemicalDependencyRecoveryBed',
+    'PEDIATRIC INTENSIVE CARE UNIT': 'PediatricIntensiveCareUnitBed',
+    'LABOR AND DELIVERY': 'LaborAndDeliveryBed',
+    'INTERMEDIATE CARE/DD': 'IntermediateCareDevelopmentallyDisabledBed',
+    'PSYCHIATRIC HEALTH': 'PsychiatricHealthBed',
+    'PEDI. DAY & RESPITE CARE': 'PediatricDayAndRespiteCareBed',
+    'DIALYSIS STATIONS': 'DialysisStationsBed',
+    'CORRECTIONAL TREATMENT CENTER': 'CorrectionalTreatmentCenterBed',
+}
+
+# start process dataset and also add one new column called 'ALicensedHealthcareFacilityCountyId'.
+with open('CA_Licensed_Healthcare_Facility_Types_And_Counts.csv', 'w', newline='') as f_out:
+    writer = csv.DictWriter(f_out, fieldnames=new_columns_1, lineterminator='\n')
+    with open('temp_data.csv', newline='') as csvfile:
+        reader = csv.DictReader(csvfile)
+        writer.writeheader()
+
+        for row_dict in reader:
+            processed_dict = {
+                'GeoId': county_to_geoID[row_dict['COUNTY_NAME']],
```

Comment:
How do we know that the county list in df2 contains all the counties in df1?  It might be true for this particular data pull.  Can we guarantee it forever?  If there is a county list in df1, it might be better to use that one.